crux-mir/README.md

```diff
@@ -56,12 +56,47 @@ versions corresponding to the last two `crux-mir` releases.
 [mir-json-readme]: https://github.com/GaloisInc/mir-json#readme
 
 
-## Setup: Build and Install
+## Setup
+
+### Build and Install
 
 Use GHC 9.6, 9.8, or 9.10. From the `crux-mir` directory, run:
 
     $ cabal install exe:crux-mir --overwrite-policy=always
 
+### Binary releases
+
+Both `crux-mir` and `mir-json` comes with nightly builds and stable binary releases. If you want to install the nightly build, do the following:
```

Comment:
It might be worth including a link to https://github.com/GaloisInc/crucible/releases for users who want stable binary releases.